### ISSUE TITLE
transformer model based on character level input, word level output code example

### ISSUE THREAD

**Issue by AyberkYavuz** [keras-team/keras-io#1283]
Hi,

I really need a sample code for my project.

In this (https://keras.io/examples/nlp/lstm_seq2seq/) example, there are character level input and output representation of data.
When character data transformed into numeric vectors, a seq2seq model trained.
 
But, in my case, I have character level input and word level output and I want to train a transformer model.

Here is a sample data of my case

typo_keyword, keyword
rkk mont, erkek mont
erkk mont, erkek mont
rkk mnt, erkek mont
akllı sat, akıllı saat
akıllı saaat, akıllı saat
aklı saat, akıllı saat

I don't want to train a character based seq2seq model (I already did and It's performance was bad at human generated test set data).

I want to train a transformer model based on character level input, word level output.

typo_keyword must be represented as character level.

keyword must be represented as word level.

Please help me!

**Comment by sampathweb** [comment]
For help with your specific use case, can you post this in https://discuss.tensorflow.org/ or Stack Overflow where community members can chime and offer suggestions?

**Comment by sampathweb** [comment]
we don't have a specific example along these lines, but KerasNLP has useful building blocks for this sort of problem...
https://keras.io/api/keras_nlp/tokenizers/byte_tokenizer/
https://keras.io/api/keras_nlp/modeling_layers/

Closing this issue, please re-open if you think its still relevant to open it here.
